<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SwagLabs-Mokhtar" failures="1" tests="10" name="Eror Validations Test" time="182.538" errors="0" timestamp="2025-07-14T22:10:29 EEST">
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="blankPasswordLoginTest" time="0.508"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="blankUsernameLoginTest" time="0.516"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="emptyFirstNameInCheckoutTest" time="3.612"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="emptyLastNameInCheckoutTest" time="148.877"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="emptyPostalCodeInCheckoutTest" time="2.29"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="invalidPasswordLoginTest" time="0.26"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="invalidUsernameLoginTest" time="0.261"/>
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="lockedOutUserLoginTest" time="0.358">
    <skipped/>
  </testcase> <!-- lockedOutUserLoginTest -->
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="lockedOutUserLoginTest" time="0.555">
    <skipped/>
  </testcase> <!-- lockedOutUserLoginTest -->
  <testcase classname="SwagLabs.Tests.ErrorValidationsTests" name="lockedOutUserLoginTest" time="0.48">
    <failure type="java.lang.AssertionError" message="expected [Epic sadface: Sorry, this user has been locked.] but found [Epic sadface: Sorry, this user has been locked out.]">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Sorry, this user has been locked.] but found [Epic sadface: Sorry, this user has been locked out.]
at SwagLabs.Tests.ErrorValidationsTests.lockedOutUserLoginTest(ErrorValidationsTests.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- lockedOutUserLoginTest -->
</testsuite> <!-- Eror Validations Test -->
